sys/include/net/gnrc/netif.h

```diff
@@ -233,6 +233,16 @@ struct gnrc_netif_ops {
     void (*msg_handler)(gnrc_netif_t *netif, msg_t *msg);
 };
 
+/**
+ * @brief   Initialize all available network interfaces.
+ *          This function is called automatically if the auto_init_gnrc_netif
+ *          module is used.
+ *          If only the gnrc_netif_init module is used instead, you can call
+ *          this function to manually set up the network interfaces at a later
+ *          time.
+ */
+void gnrc_netif_auto_init(void);
```

Comment:
`gnrc_netif_init()` or `gnrc_netif_init_devs()`?

Reply:
`gnrc_netif_init_devs()` is better.